templates/vars.yml.j2

```diff
@@ -181,3 +167,29 @@ matrix_nginx_proxy_proxy_synapse_additional_server_configuration_blocks:
   auth_basic "protected";
   auth_basic_user_file /nginx-data/matrix-synapse-metrics-htpasswd;
   }'
+
+
+# Synapse configuration extension yaml
+
+matrix_synapse_configuration_extension_yaml: |
+{%- set connection_pool_min = CP_MIN|default('5') %}
```

Comment:
Could we use more descriptive names for the env vars passed? Not sure if I'm gonna remember the `CP_MIN/MAX` meaning at a first glance after this.